### ISSUE TITLE
Complete inside docstrings?

### ISSUE THREAD

**Issue by dpo** [JunoLab/atom-latex-completions#24]
For some reason, completions don't occur inside Julia docstrings for me. I logged the cursor scope while inside a docstring and accordingly added
```
.source.julia, string.docstring.julia, .source.gfm
```
to the config of this package, but with no luck. I also tried `.docstring.julia` instead of `string.docstring.julia` but with no success.
Completions happily occur outside of docstrings.

Any ideas?

**Comment by pfitzseb** [comment]
Hm, I can't reproduce this -- latex completions work just fine in docstrings and strings for me.

**Comment by dpo** [comment]
Since I posted I noticed that the issue I'm having is more nuanced. Say I open a docstring with triple quotes and immediately type `\lambda`:
```julia
"""\lambda
```
then completion kicks in correctly. But if I type `\lambda` on the next line:
```julia
"""
\lambda
```
completion doesn't occur. Do you not observe the same?

**Comment by pfitzseb** [comment]
No, both cases work fine for me. What are your `Selector` and `Disable For Selector` settings?

**Comment by dpo** [comment]
I restored them to the default values.

**Comment by joshday** [comment]
Do you have autocomplete-bibtex installed?  At .source.gfm, all other autocompletions get disabled.

https://github.com/apcshields/autocomplete-bibtex/issues/72
